ratelimit/decorators.py

```diff
@@ -72,7 +76,9 @@ def wrapper(*args, **kargs):
                 # If the number of attempts to call the function exceeds the
                 # maximum then raise an exception.
                 if self.num_calls > self.clamped_calls:
-                    raise RateLimitException('too many calls', period_remaining)
+                    if self.raise_on_limit:
+                        raise RateLimitException('too many calls', period_remaining)
+                    return lambda *args, **kwargs: None
```

Comment:
I want to avoid raising exception when reaching the limit. but, if i'll just skip the exception i'll eventually get to function call (f(*args, **kwargs)). This will make the rate limiter useless. I want to avoid exception but still, calling the function only as much time as i allowed by rate limiter.
Since decorator suppose to return a callable - i return this lambda.

Reply:
I'm still not sure I entirely agree with having to return a callable. For instance this line (https://github.com/tomasbasham/ratelimit/pull/18/files#diff-413b58119aa3f7e639d2a9131a99199eR83) does not return a callable from the decorator, it returns the result of a function call (the decorated function).

Reply:
I think you are right. I wasn't sure about it myself... I will change it to
```python
return None
```

Reply:
I think simply

```python
return
```

Reply:
Same same...
```return``` is equivalent to ```return None```.